frank-doc-frontend/src/app/views/home/home-component-list/home-component-list.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="tabs">
+  <div class="tab active">
+    <span>Components</span>
+    <span class="count">{{ components.length }}</span>
+  </div>
+  @if (showRelated) {
+    <div class="tab">
+      <span>Related</span>
+      <span class="count">{{ relatedComponents.length }}</span>
+    </div>
+  }
+  <div class="list-modes"></div>
+</div>
+<div class="components">
+  @for (component of components; track component.item.name) {
+    <div class="card">
+      <p class="label">{{ component.item.labels ? getFirstLabel(component.item.labels) : '' }}</p>
+      <p class="name" [title]="component.item.name">{{ component.item.name }}</p>
+      <p class="element-names" [title]="component.item.elementNames.join('\n')">
+        @for (elementName of component.item.elementNames; track elementName; let first = $first) {
+          @if (!first) {
+            &bull;
+          }
+          <span>{{ elementName }}</span>
+        }
+      </p>
+      <p class="description">{{ component.item.description | truncate: 150 }}</p>
+      <ff-button
+        [routerLink]="[
+          '/',
+          getFirstFilter(component.item.labels),
+          getFirstLabel(component.item.labels),
+          component.item.name,
+        ]"
+        >-></ff-button
```

Comment:
This is a nice character to use instead of 2, but according to another comment you would rather have an html entity, I assume. I found this `&rarr;` (→) as possible option but maybe you can find a better one?

Reply:
Oh, you are completely right. It might be best to even use a real icon for this.